### ISSUE TITLE
Screen readers fail to report the items in the list of added audio/video filters

### ISSUE THREAD

**Issue by k-kolev1985** [obsproject/obs-studio#11400]
### Operating System Info

Windows 11

### OBS Studio Version

30.2.3

### OBS Studio Log URL

https://obsproject.com/logs/niO15nP3SIxPJp7D

### Expected Behavior

The screen reader should report the filter's name or if it is enabled/disabled.

### Current Behavior

The screen reader does not report the filter's name or if it is enabled/disabled.

### Steps to Reproduce

1.
2.
3.
...
1. Launch a screen reader like NVDA or Narrator.
2. Launch OBS Studio.
3. Create a scene and add at least one source to it.
4. From the context menu for the added source, select "Filters".
5. In the "Filters" dialog/window, add a few filters.
6. Move the focus to the list of filters and use up/down arrow to move between the filters in the list.

### Anything else we should know?

This is an accessibility bug with OBS Studio. Screen readers like NVDA and Narrator fail to read the items in the list of added audio/video filters. The focus is in the list, pressing up/down arrow moves between the added filters, but screen readers do not read the name of the selected filter and if it is enabled or disabled.

Can the problem be fixed? The problem was partially fixed some time ago for the "Sources" list in the main OBS Studio window. I say "partially", because screen readers now report the name of each source, but not if it is enabled or not (but the last one should go into another report, probably).
